internal/controller/operator/converter/v1alpha1/apis.go

```diff
@@ -143,6 +143,7 @@ func ConvertScrapeConfig(promscrapeConfig *promv1alpha1.ScrapeConfig, conf *conf
 	cs.Annotations = converter.FilterPrefixes(promscrapeConfig.Annotations, conf.FilterPrometheusConverterAnnotationPrefixes)
 	cs.Spec.RelabelConfigs = converter.ConvertRelabelConfig(promscrapeConfig.Spec.RelabelConfigs)
 	cs.Spec.MetricRelabelConfigs = converter.ConvertRelabelConfig(promscrapeConfig.Spec.MetricRelabelConfigs)
+	cs.Spec.Path = *promscrapeConfig.Spec.MetricsPath
```

Comment:
I think, it's better to use ptr.Deref, otherwise operator will crash with panic at null metricsPath

Reply:
See failed tests

Reply:
updated